docs/docs/workflows/update-manifest.md

```diff
@@ -52,17 +52,123 @@ Don't define other values except the `SecurityVersion` value for a package, as M
 ```
 
-On success, no message will be returned and your MarbleRun logs should highlight that an update manifest has been set. On error, the API endpoint will return an error message. If you receive `unauthorized user`, MarbleRun either received no client certificate over the TLS connection, or you used the wrong certificate.
+On success, no message will be returned and your MarbleRun logs should highlight whether the update manifest has been set or further acknowledgements are required. On error, the API endpoint will return an error message. If you receive `unauthorized user`, MarbleRun either received no client certificate over the TLS connection, or you used the wrong certificate.
+
+## Acknowledging an update
+
+Use the CLI to acknowledge a full manifest update, specifying the client certificate and private key of a user with appropriate permissions:
+
+```bash
+marblerun manifest update update-manifest.json $MARBLERUN --cert=user-cert.pem --key=user-key.pem --acknowledge --era-config=era.json
+```
+
+### Example: Multi-party update
+
+The following gives an example of a full manifest update with multiple parties:
+
+Assume the following `Users` and `Roles` were defined in the Manifest:
+
+```javascript
+    "Users": {
+        "alice": {
+            "Certificate": "-----BEGIN CERTIFICATE-----\nMIIFZTCCA02gAwIBAgIUANHwS8RM0PUDl9htA+yWJx9WqucwDQYJKoZIhvcNAQEL\nBQAwQjELMAkGA1UEBhMC=\n-----END CERTIFICATE-----\n",
+            "Roles": [
+                "UpdateManifest"
+            ]
+        },
+        "bob": {
+            "Certificate": "-----BEGIN CERTIFICATE-----\nMIIFZTCCA02gAwIBAgIUJvtF7KRsunTmWVtpU9198HUxyLEwDQYJKoZIhvcNAQEL\nBQAwQjELMAkGA1UEBhMC=\n-----END CERTIFICATE-----\n",
+            "Roles": [
+                "UpdateManifest"
+            ]
+        }
+    },
+    "Roles": {
+        "UpdateManifest": {
+            "ResourceType": "Manifest",
+            "Actions": ["UpdateManifest"]
+        }
+    },
+```
+
+1. Alice applies an update by uploading an update manifest:
+
+    ```bash
+    $ marblerun manifest update apply new_manifest.json $MARBLERUN ---cert alice.crt --key alice-private.pem
+    Coordinator verified 
+    Loading client certificate
+    Creating client
+    Successfully verified Coordinator, now uploading manifest
+    Update manifest set successfully
+    ```
+
+    The Coordinator log shows the following:
+
+    ```bash
+    {"level":"info","ts":1674205619.1967707,"caller":"clientapi/clientapi.go:199","msg":"UpdateManifest called"}
+    {"level":"info","ts":1674205619.2007706,"caller":"clientapi/clientapi.go:282","msg":"UpdateManifest successful. Waiting for acknowledgments to apply the update","missingAcknowledgments":1}
+    ```
+
+2. Bob checks the planned update:
+
+    ```bash
+    $ marblerun manifest update get $MARBLERUN
+    Successfully verified Coordinator
+    {
+        "Manifest": {
+            "Marbles": {
+                ...
+    }
```

Comment:
```suggestion
    $ marblerun manifest update get $MARBLERUN
    Successfully verified Coordinator
    {
        "Marbles": {
                ...
    }
```
https://github.com/edgelesssys/marblerun/pull/356 updates the command to return the exact bytes of the pending manifest